setup.sh

```diff
@@ -72,6 +72,7 @@ pip install --upgrade -r requirements.txt
 
 # wordnet
 python -m nltk.downloader -d jarviscli/data/nltk wordnet
+pip install numpy
```

Comment:
Should be added to requirements.txt

=> Just add 'numpy' (alphabetically order) to requirements.txt file and and ``pip install --upgrade -r requirements.txt`` in setup.sh will install everything listed there.